src/views/CartView.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="cart-view" class="row">
     <div class="col">
-      <h3>{{ 'lifelines-webshop-cart-header' | i18n }}</h3>
+      <h3>{{$t('lifelines-webshop-cart-header')}}</h3>
```

Comment:
why not keep the filter ?

Reply:
It makes translations easier to spot; e.g. the i18n test looks for method calls like these.